.github/workflows/ci.yml

```diff
@@ -63,9 +59,26 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 16
-      - name: Install NPM dependencies
+      - name: Install npm dependencies
         run: npm ci
       - name: Lint code
         run: npm run lint
       - name: Build
         run: npm run compile
+
+  integration:
+    runs-on: ubuntu-latest
+    permissions: read-all
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get versions
+        run: |
+          docker-compose --version;
+          docker --version;
```

Comment:
Could omit it?

Reply:
Could. I thought about that. Proved to be useful during my dev. Harms no one.

Do you think I should?